### ISSUE TITLE
GU16 and new DLC Rise new world

### ISSUE THREAD

**Issue by Talagad** [xforce/anno1800-mod-loader#180]
Hi,

after game update the game are not starting with active mod loader.

Regards
Talagad

**Comment by Talagad** [comment]
here the information from the eventlog:
---
Name der fehlerhaften Anwendung: Anno1800.exe, Version: 18.10094.1.46856, Zeitstempel: 0x6387838d
Name des fehlerhaften Moduls: ntdll.dll, Version: 10.0.22621.755, Zeitstempel: 0x8a328c67
Ausnahmecode: 0xc0000005
Fehleroffset: 0x00000000000a3db7
ID des fehlerhaften Prozesses: 0x0x18E8
Startzeit der fehlerhaften Anwendung: 0x0x1D90B11F229C0F1
Pfad der fehlerhaften Anwendung: D:\Spiele\Anno 1800\Bin\Win64\Anno1800.exe
Pfad des fehlerhaften Moduls: C:\WINDOWS\SYSTEM32\ntdll.dll
Berichtskennung: d45c3f4e-3613-44be-92dc-d941a0be0a2f
Vollständiger Name des fehlerhaften Pakets: 
Anwendungs-ID, die relativ zum fehlerhaften Paket ist:
---

sorry for german but I have a german OS.

Regards

**Comment by arch7** [comment]
`Faulting application name: Anno1800.exe, version: 18.10094.1.46856, time stamp: 0x6387838d
Faulting module name: ntdll.dll, version: 10.0.19041.2130, time stamp: 0xb5ced1c6
Exception code: 0xc0000005
Fault offset: 0x0000000000063526
Faulting process id: 0x3e44
Faulting application start time: 0x01d90b13dca15ccf
Faulting application path: C:\Games\Anno 1800\Bin\Win64\Anno1800.exe
Faulting module path: C:\WINDOWS\SYSTEM32\ntdll.dll
Report Id: ed9984b3-d4e9-427a-8fe7-9f5f174c3dcd
Faulting package full name: 
Faulting package-relative application ID: `

`Fault bucket , type 0
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: Anno1800.exe
P2: 18.10094.1.46856
P3: 6387838d
P4: ntdll.dll
P5: 10.0.19041.2130
P6: b5ced1c6
P7: c0000005
P8: 0000000000063526
P9: 
P10: 

Attached files:

These files may be available here:
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportQueue\AppCrash_Anno1800.exe_fc90a6728e6142ae7698f3848a19785691b22c2_3903fefe_14e6efe4-7529-4ea3-9a19-d8b9b99735dd

Analysis symbol: 
Rechecking for solution: 0
Report Id: ed9984b3-d4e9-427a-8fe7-9f5f174c3dcd
Report Status: 4
Hashed bucket: 
Cab Guid: 0`

`Fault bucket 1206694638675026184, type 4
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: Anno1800.exe
P2: 18.10094.1.46856
P3: 6387838d
P4: ntdll.dll
P5: 10.0.19041.2130
P6: b5ced1c6
P7: c0000005
P8: 0000000000063526
P9: 
P10: 

Attached files:
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER91F7.tmp.WERInternalMetadata.xml

These files may be available here:
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_Anno1800.exe_fc90a6728e6142ae7698f3848a19785691b22c2_3903fefe_14e6efe4-7529-4ea3-9a19-d8b9b99735dd

Analysis symbol: 
Rechecking for solution: 0
Report Id: ed9984b3-d4e9-427a-8fe7-9f5f174c3dcd
Report Status: 268435456
Hashed bucket: 0c8320f87d300f7d10bf0a616a97e108
Cab Guid: 0`

**Comment by jasonbuff** [comment]
i had to verify the game to get it to start but will have to wait on the bootloader to be updated for the mods to work again

**Comment by ascheff94** [comment]
I have the same issue as the op. Played the game just minutes before, so im confident it is related to the GU16.

**Comment by Mannie313** [comment]
ich hatte es auch kurz zuvor noch mit mods gespielt und jetzt geht nix mehr. die beiden original logs sind leer

**Comment by benjamincere** [comment]
same issue here, I just wanted to restart to play with the dlc and nothing works. I have to repair the game to restart (without mods)

**Comment by Otmarius** [comment]
Stimmt. Mit Mod-Loader startet Anno 1800 nicht bzw. schließt sich sofort wieder, mit der Original Phyton35.dll wird es gestartet.

**Comment by xforce** [comment]
I'm working on it. Just got off work.

**Comment by jasonbuff** [comment]
Your the best God bless you and your family the chosen one

**Comment by Talagad** [comment]
@xforce; Many thanks for your work!

**Comment by Gunslinger32** [comment]
Thank you so much, hope you can solve this.

**Comment by Otmarius** [comment]
xforce, you are my hero. Thank you

**Comment by supra3turbo** [comment]
Ive just had the exact same problem the python files for loader no longer work, i will make sure i keep my eye on this post ive been trying to get it working for an hour and a half and this is the first time ive found anything mentioned about the update not working, really looking forward for it to be working again. Thanks :)

**Comment by xforce** [comment]
> Ive just had the exact same problem the python files for loader no longer work, i will make sure i keep my eye on this post ive been trying to get it working for an hour and a half and this is the first time ive found anything mentioned about the update not working, really looking forward for it to be working again. Thanks :)
> 
> (will this forum give me a notification when this has been fixed? thanks :)

There should be a button on the right side of the screen where you can subscribe to get notified when things happen in this issue @supra3turbo

**Comment by xforce** [comment]
Fixed in [0.9.3](https://github.com/xforce/anno1800-mod-loader/releases/tag/v0.9.3)

**Comment by supra3turbo** [comment]
ITS FIXED :D

**Comment by WaschBaerHH** [comment]
thank you, great job

**Comment by Totally-Angry** [comment]
you are awesome!

**Comment by magicalcookie** [comment]
There was a somewhat critical issue in 0.9.3 that caused external files (like new textures) to not load. This has been fixed in [0.9.4](https://github.com/xforce/anno1800-mod-loader/releases/tag/v0.9.4)

**Comment by jimz011** [comment]
@xforce not sure if I should open a new issue for this (since it states pre-release on your release). But after installing the pre-release the modloader works again and Anno starts fine. The mods also seem to work fine, however there are grid lines showing everywhere. At first I thought it was just in my head, but I am pretty sure they weren't there before this release.

![image](https://user-images.githubusercontent.com/47046878/206592146-31a21f51-e17c-441c-acdf-adefe493e9fa.png)

If you look at the screenshot carefully you will see grid lines (squares) on the map. They are even visible in the sea
![image](https://user-images.githubusercontent.com/47046878/206592301-67308f4d-c96b-438a-a721-6e8570006bc9.png)

Not a big deal, but I thought you might want to know about it 😉

**Comment by jimz011** [comment]
Addition to my last post:

It also is apparent in the menu's and loading screens:
![image](https://user-images.githubusercontent.com/47046878/206593228-04f99992-1c7c-4e60-b2e7-38995f34e428.png)

It is very hard to see, but they are there.

Lastly, all the service buildings also have disappeared (e.g. firestation, policestation and hospital)

**Comment by jimz011** [comment]
Oh.... The 0.9.4 release came shortly after, I have updated it. The lines are gone! Nice!

The service buildings however are still missing (the ones already built are gone and there is no way to build new ones).
To be sure it wasn't some other mod I disabled all the mods and started the game.

**Comment by jasonbuff** [comment]
Same results

**Comment by KagamineLen2333** [comment]
I just purchased a steam version. Using 0.9.4 on the steam version, crashed almost every 5 mins at most...try older version (0.9.2 ) on steam, game won't start
this never happened in ubi version. I could build service buildings.

**Comment by jasonbuff** [comment]
I'd hate to go through each mod if it's a loader problem I will try
tomorrow I ha e so many mods though

**Comment by badlands06** [comment]
Hi all,
just installed 0.9.4. Game starts, I'm able to select my latest saved game but it crashes a few seconds after I select it. Anyone else has the same issue?

**Comment by tefracky** [comment]
> Hi all, just installed 0.9.4. Game starts, I'm able to select my latest saved game but it crashes a few seconds after I select it. Anyone else has the same issue?

See https://github.com/xforce/anno1800-mod-loader/issues/183

**Comment by badlands06** [comment]
> > Hi all, just installed 0.9.4. Game starts, I'm able to select my latest saved game but it crashes a few seconds after I select it. Anyone else has the same issue?
> 
> See #183

Thanks tefracky!
Will keep an eye on #183

**Comment by lcyril49** [comment]
> Mais après avoir installé la pré-version
ou ce trouve Mais après avoir installé la pré-version ? apres avoir mis dll sa plante

**Comment by popojk454545** [comment]
> 

yes
